_includes/sql/v1.1/functions.md

```diff
@@ -150,6 +150,17 @@ Function &rarr; Returns | Description
 <code>translate(input: <a href="string.html">string</a>, find: <a href="string.html">string</a>, replace: <a href="string.html">string</a>) &rarr; <a href="string.html">string</a></code> | <span class="funcdesc">In `input`, replaces the first character from `find` with the first character in `replace`; repeat for each character in `find`. <br/><br/>For example, `translate('doggie', 'dog', '123');` returns `1233ie`.</span>
 <code>upper(val: <a href="string.html">string</a>) &rarr; <a href="string.html">string</a></code> | <span class="funcdesc">Converts all characters in `val`to their to their upper-case equivalents.</span>
 
+### Array Functions
```

Comment:
@justinj Yes, I am trying to figure out the right place for it as well. Let me think about it a bit more.